src/parent/reducers/reducers.js

```diff
@@ -1,10 +1,33 @@
-import { combineReducers } from 'redux';
+import { PARENT as ACTION_TYPES } from '../../shared/constants/actionTypes';
 
 import childWindows from './childWindows';
 
-const rootReducer = combineReducers({
-    childWindows
-});
+function rootReducer(state = { childWindows: {}, dragOut: null }, action) {
```

Comment:
Can we keep `combineReducers` and have a `dragOut` reducer? See https://github.com/ScottLogic/StockFlux/pull/863#discussion_r72661223

I'm not sure, but I think `draggedOutStockCode` might be a bit more descriptive than `dragOut`?